packages/angular/src/generators/application/files/base/tsconfig.json__tpl__

```diff
@@ -8,12 +8,6 @@
   "references": [
     {
       "path": "./tsconfig.app.json"
-    },
-    <% if(unitTesting) { %>{
-      "path": "./tsconfig.spec.json"
-    },<% } %>
-    {
-      "path": "./tsconfig.editor.json"
```

Comment:
Adding the `tsconfig.editor.json` is handled by the generator's code to ensure it is the last reference.